src/views/Login.vue

```diff
@@ -112,3 +136,15 @@
   })
   export default class Login extends Vue {}
 </script>
+<style scoped lang="scss">
+  .create-account {
+    color: #009688;
+    border-color: #009688;
+    background: #ffffff;
+  }
+  .create-account:hover {
+    color: #009688;
+    border-color: #009688;
+    background: #ffffff;
+  }
+</style>
```

Comment:
Can you elaborate on why there are some styles in the scoped component's css section and some in the main css file. I think you can choose one or the other. In the present case I would prefer if you put it in the `main.scss` file as there already is some code related to the login page there. If you needed the `scoped` feature, you can always increase specificity of the selectors in the `main.scss` file by nesting them.

Reply:
these Css lines are specifically made to be used inside are composed,  for me it makes more sense to put them here. 
as explained in the Vuejs style guide : https://v2.vuejs.org/v2/style-guide/?redirect=true#Component-style-scoping-essential
but it is true that it would be more consistent with what we have already done in the past on this project.

Reply:
I understand what you're saying, and I agree on the conceptual side, I would have done it the way you are talking about. But if at some point we head to a refactoring, it will be way easier to extract the code from the same place. Thanks for your understanding.

Reply:
@seegwen Thank you for the review, i think i have made all the requested changes